gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/tasks/CreateManifestTaskIntegrationSpec.groovy

```diff
@@ -27,6 +29,16 @@ import spock.lang.Unroll
 
 class CreateManifestTaskIntegrationSpec extends GradleIntegrationSpec {
```

Comment:
should we pull the tests for the product dependency discovery out into a seperate test suite? it might be easier to mock and verify the behaviour we want

Reply:
yes, in the process of doing so